riptable/rt_merge.py

```diff
@@ -2063,12 +2063,12 @@ def merge_indices(
 #        assert left_fancyindex_rowcount == right_fancyindex_rowcount,\
 #            f'Left and right rowcount differ ({left_fancyindex_rowcount} vs. {right_fancyindex_rowcount}).'
 
-    # DEBUG: Print the constructed indices.
-    if logger.isEnabledFor(logging.DEBUG):
-        left_fancyindex = join_indices.left_index
-        right_fancyindex = join_indices.right_index
-        logger.debug(f'left_fancyindex ({type(left_fancyindex)}): {left_fancyindex}')
-        logger.debug(f'right_fancyindex ({type(right_fancyindex)}): {right_fancyindex}')
+#    # DEBUG: Print the constructed indices.

```

Comment:
It could probably be deleted. I thought about doing that, but also considered putting it behind some feature-flag to enable it in case there's some issue with merging (e.g. the function produces bad results). I'll think about it a bit more and circle back to take one of those two actions later (vs. just leaving it commented out forever).